common/src/benchmarking.rs

```diff
@@ -150,6 +155,7 @@ pub fn set_program(program_id: H256, code: Vec<u8>, static_pages: WasmPageNumber
             static_pages,
             state: ProgramState::Initialized,
             gas_reservation_map: GasReservationMap::default(),
+            block_number,
```

Comment:
for benchmarking module we can use here just default

Reply:
Done